docs/uplink/become-a-provider.md

```diff
@@ -133,7 +133,8 @@ The Helm chart is called *inlets-uplink-provider*, you can deploy it using the c
 
 ```bash
 helm upgrade --install inlets-uplink \
-  oci://ghcr.io/inlets/inlets-uplink-provider \
+  oci://ghcr.io/openfaasltd/inlets-uplink-provider \
+  --version 0.1.9 \
```

Comment:
0.2.0 now I think?

Reply:
I removed the version flag so the latest chart will be used when copying the helm command.